### ISSUE TITLE
`r5py.sampledata` namespace with `r5py.sampledata.saopaulo` and `r5py.sampledata.helsinki` sub-packages

### ISSUE THREAD

**Issue by christophfink** [r5py/r5py#347]
For the sake of clarity in the documentation, and for making data retrieval in workshops easier, we should create sub-packages that contain the sample data (or, rather, download them on the fly on first use, both PyPi and conda have limitations when it comes to package size)

Also see discussion between @HTenkanen and me on our slack

**Comment by christophfink** [comment]
relates to #345

**Comment by christophfink** [comment]
I’m thinking whether we should implement this right away, as it’s probably not so much more work than compiling a downloadable zip file

**Comment by HTenkanen** [comment]
@christophfink I think it would be great, if we manage to do this right away! 👍🏻 I guess you had a good idea how to do this?

**Comment by christophfink** [comment]
Yes, I have it in my head, but won’t be able to put it into code until earliest tomorrow (but I guess the other issues for v0.1.1 need a bit of work, too)

**Comment by christophfink** [comment]
See the changes merged in #348 and the new repositories https://github.com/r5py/r5py.sampledata.helsinki and https://github.com/r5py/r5py.sampledata.sao_paulo
Now still removing the data from the core repository, and adapting the documentation to use the sample data packages
